index.js

```diff
@@ -32,6 +37,50 @@ module.exports = {
         ],
       },
     },
+    delete: {
+      validation: {
+        function: 'checkboxValidation',
+        on: 'change',
+      },
+      script: {
+        function: 'fileScript',
+        on: 'change',
+      },
+      type: 'checkbox',
+      label: 'Delete file',
+      options: [
+        { label: 'Delete file',
+          value: 'true',
+        },
+      ],
+    },
   },
-  html: '<label for="{{file.id}}">{{file.label}}</label><input type="{{file.type}}" id="{{file.id}}" name="{{file.name}}" value="{{file.value}}" {% if settings.types %}{%set comma = joiner() %}accept="{% for type in settings.type %}{{comma()}}{{type}}{% endfor %}{% endif %}" />',
+  html: `<div class="file">
+    <label for="{{file.id}}">{{file.label}}</label>
+    <input type="{{file.type}}" id="{{file.id}}" name="{{file.name}}" value="{{file.value}}" class="file--upload" {% if settings.types %}{%set comma = joiner() %}accept="{% for type in settings.type %}{{comma()}}{{type}}{% endfor %}{% endif %}" />
```

Comment:
Why does this have a class but not its label? It's OK either way, but I'd prefer not to have the inconsistency between the two